front/components/vaults/VaultResourcesList.tsx

```diff
@@ -87,9 +87,7 @@ const getTableColumns = ({
   const managedByColumn = {
     header: "Managed by",
     accessorFn: (row: RowData) =>
-      (row.dataSourceView.kind === "default"
-        ? row.dataSourceView.dataSource.editedByUser?.imageUrl
-        : row.dataSourceView.editedByUser?.imageUrl) ?? "",
+      row.dataSourceView.dataSource.editedByUser?.imageUrl ?? "",
```

Comment:
We should use the editedByUser of the data source view, not the data source ..?

Reply:
Agree, but have no idea why this isn't properly set.. or maybe it's just something broken on my local

Reply:
Will check more accurately on Monday